nservicebus/azure-service-bus/configuration/configuration.md

```diff
@@ -0,0 +1,200 @@
+ * `PrefetchCount(int)`: This setting will make the receiver prefetch messages on receive operations, and acts like client side batching. Defaults to 200, meaning that each receive operation will pull in another 200 messages.
+ * `RetryPolicy(RetryPolicy)`: Determines how the receiver should respond on transient connectivity failures. Defaults to `RetryPolicy.Default`, which is an exponential retry under the hood.
+ * `AutoRenewTimeout(TimeSpan)`: When using `ReceiveMode.PeekLock`, the broker will lock a received message for a period specified as `LockDuration(TimeSpan)` on the entity. If processing doesn't end within this time period, a message will reappear on the entity and will be reprocessed. For long running operations this might imply that the operation will be executed multiple times in parallel, which is usually undesirable. This setting allows to extend the lock automatically when processing hasn't finished yet. Ensure that the `TimeSpan` specified here is a little smaller than the `TimeSpan` specified as `LockDuration`.
+
+### Message Senders
+
+The following settings determine how NServiceBus will send messages to Azure Service Bus entities.
+
+ * `RetryPolicy(RetryPolicy)`: Determines how the sender should respond on transient connectivity failures. Defaults to `RetryPolicy.Default`, which is an exponential retry under the hood.
+ * `BackOffTimeOnThrottle(TimeSpan)`: 'As A Service' products will tell there clients to back off under heavy load and so will Azure Service Bus. This setting specifies how long NServiceBus will wait to try sending again in these occurances. Defaults to 10 seconds. Note, by default this backoff period will increase exponentially during retries. 
+ * `RetryAttemptsOnThrottle(int)`: Defines how many times the transport should attempt to retry a send operation when being operated before throwing an exception.
+ * `MaximumMessageSizeInKilobytes(int)`: Specifies how large messages (and batches of messages) can be. This can differ between different types of namespaces. The default value of 256 aligns with Standard namespaces.
+ * `MessageSizePaddingPercentage(int)`: As specified in the [msdn documentation](https://msdn.microsoft.com/en-us/library/microsoft.servicebus.messaging.brokeredmessage.size.aspx), the Size property on a BrokeredMessage object will only provide accurate values after sending a message, which turns it pretty useless to compute a maximum batch size for batching. Therefore NServiceBus implements it's own batch size computation. Part of this computation requires a guesstimate of how much overhead will be caused by serialization of the brokered message properties. This percentage represents that guesstimate and defaults to 5%.
```

Comment:
How about 

The Size property on a `BrokeredMessage` will provide accurate values only after the message was sent (refer to the [BrokeredMessage.Size Property](https://msdn.microsoft.com/en-us/library/microsoft.servicebus.messaging.brokeredmessage.size.aspx) documentation for more details). Therefore it can't be used to compute an accurate maximum batch size. NServiceBus has it's own batch size computation implementation, which requires providing an estimate for how much overhead (in %) will be caused by the serialization of the brokered message properties. The estimate is specified using `MessageSizePaddingPercentage`. Defaults to 5%.

> in %

of what? total sending time?